otter/convergence/steps.py

```diff
@@ -73,6 +80,12 @@ def set_server_name(server_config_args, name_suffix):
     return set_in(server_config_args, ('server', 'name'), name)
 
 
+def append_stack_uuid(stack_config, uuid):
```

Comment:
I'll leave it because I like the small bit of clarity it adds and that it matches create server's `set_server_name`.

Also, added docstring.